addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -2471,6 +2472,10 @@ export class OdooEditor extends EventTarget {
                 this._selectTableCells(range);
                 appliedCustomSelection = true;
             }
+        } else if (selectedCells.length === 1 && selectedCells[0] === startTd) {
+            // The selection goes through a single cell -> select the cell.
+            this._selectTableCells(range);
+            appliedCustomSelection = true;
```

Comment:
@Zynton we need this condition in case when we select single cell and try to apply background color in that case the `startTd` and `endTd` will be same and we don't have `ev` and it doesn't pass any condition.So,it closes the toolbar.So, we need this condition as it will help toolbar to stay opened.

The modifications in `wysiwyg.js` file is for firefox.When we select multiple cell and apply background color then it shifts selection to last two cell.So, for that case i made changes in `wysiwyg.js` file.